changelog.d/7497.bugfix

```diff
@@ -0,0 +1 @@
+Omit displayname or avatar_url if they aren't set instead of returning null.
```

Comment:
Up to you if you include the attribution, but we'd encourage you to!

```suggestion
When sending `m.room.member` events, omit `displayname` and `avatar_url` if they aren't set instead of setting them to `null`. Contributed by Aaron Raimist.
```